exodus_lambda/functions/base.py

```diff
@@ -14,5 +14,14 @@ def conf(self):
 
         return self._conf
 
+    @property
+    def region(self):
+        # Use specified region if among available regions.
+        # Otherwise, default to first available region listed.
+        table_conf = self.conf["table"]
+        if table_conf["region"] in table_conf["available_regions"]:
```

Comment:
I don't think this is correct:

It's looking at config["table"]["region"].  The value in the config file template in the repo is $AWS_REGION.  We expect that value to be replaced with a real region while building the deployment package for our lambdas, right?

However, there is only *one* deployment package which is used to run the lambda in *multiple* regions so it can't possibly be correct for all regions.

I think "region" should disappear from the config file entirely and should be replaced with os.environ.get("AWS_REGION").

Reply:
We need the config file to handle any environment variables, as Lambda@Edge won't accept them.

Reply:
I don't think that's right.  I think the restriction on Lambda@Edge is that you can't define your own environment variables in the typical way as done for lambdas, but AWS_REGION and maybe some other built-in env vars provided by the platform are still usable.  I've dumped os.environ in a lambda before and it contained AWS_REGION.

In the issue description, I also linked blog post https://aws.amazon.com/blogs/networking-and-content-delivery/leveraging-external-data-in-lambdaedge/ which shows using AWS_REGION environment variable from a lambda.

Reply:
That's right. I hadn't considered Lambda having built-in environment variables.